core-plugins/src/main/java/co/cask/hydrator/plugin/transform/ProjectionTransform.java

```diff
@@ -125,7 +125,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
   @Override
   public void initialize(TransformContext context) throws Exception {
     super.initialize(context);
-    init();
+    init(null);
```

Comment:
Yes. init() is called both from configurePipeline to validate and initialize to initialize the variables with the inputs.